modules/aws_k8s_base/tf_module/nvidia_daemonset.tf

```diff
@@ -1,15 +1,35 @@
 resource "helm_release" "nvidia_daemonset" {
+  namespace       = "kube-system"
   chart           = "nvidia-device-plugin"
   name            = "nvidia-device-plugin"
   repository      = "https://nvidia.github.io/k8s-device-plugin"
-  version         = "0.9.0"
+  version         = "0.14.0"
   atomic          = true
   cleanup_on_fail = true
   values = [
     yamlencode({
-      nodeSelector : {
-        gpu : "true"
-      }
+      affinity : {
+        nodeAffinity : {
+          requiredDuringSchedulingIgnoredDuringExecution : {
+            nodeSelectorTerms : [
+              {
+                matchExpressions : [
+                  {
```

Comment:
“use_gpu” sets the correct ami type: https://github.com/unionai/opta/blob/2f6cbfce158c90a18d82eab3f06b736fac5a1275/modules/aws_nodegroup/aws_nodegroup.py#L20

but, we could consider updating to ami_type across the board eventually to avoid the deprecation warning. it should just be a config change with no TF diff, iirc. but since we control the fork, the risk of deprecation is non-existent.